### PR TITLE
Account deletion at Mercado Libre and Viva o Linux

### PR DESCRIPTION
**Pull request by SnwMds**
<h3 align='center'>Changes</h3>

* **Updated:** `difficulty`, `notes` and `notes_pt_br` of the "Mercado Libre Uruguay" account deletion info
* **Added:** "Mercado Libre Argentina" account deletion info
* **Added:** "Mercado Libre Bolivia" account deletion info
* **Added:** "Mercado Libre Chile" account deletion info
* **Added:** "Mercado Libre Colombia" account deletion info
* **Added:** "Mercado Libre Costa Rica" account deletion info
* **Added:** "Mercado Libre Ecuador" account deletion info
* **Added:** "Mercado Libre Guatemala" account deletion info
* **Added:** "Mercado Libre Honduras" account deletion info
* **Added:** "Mercado Libre México" account deletion info
* **Added:** "Mercado Libre Nicaragua" account deletion info
* **Added:** "Mercado Libre Panamá" account deletion info
* **Added:** "Mercado Libre Paraguay" account deletion info
* **Added:** "Mercado Libre Perú" account deletion info
* **Added:** "Mercado Libre República Dominicana" account deletion info
* **Added:** "Mercado Libre Salvador" account deletion info
* **Added:** "Mercado Libre Uruguay" account deletion info
* **Added:** "Mercado Libre Venezuela" account deletion info
* **Added:** "Viva o Linux (VOL)" account deletion info

<h3 align='center'>Additional information</h3>

The "Mercado Pago" service isn't available in some countries. This is why domains associated with the "Mercado Pago" aren't present in some account deletion entries. In this case, only the domains related to "Mercado Libre" are present.

### CODE REVIEW

**Comment by tupaschoal** [comment]
@Nightfirecat please tell me if you'll be reviewing or don't find necessary, I intend to squash all MercadoLibre commits.

File: _data/sites.json [hunk 1]
```diff
@@ -4415,16 +4415,200 @@
         ]
     },
 
+    {
+        "name": "Mercado Libre Argentina",
+        "url": "http://myaccount.mercadolibre.com.ar/cancelar-cuenta",
+        "difficulty": "medium",
+        "notes": "To cancel/delete your account, access the account deletion link specified above and log in to your account. In the options, select a reason why you want to cancel your account, click in the checkbox \"Estoy de acuerdo\", and confirm your action by clinking \"Aceptar\".</br></br><strong>Warning:</strong> You may cancel your account only if there is no limitation, partial and/or total blockage applied to it. You will also be unable to cancel your account if you have outstanding payments and/or purchases. Mercado Libre and Mercado Pago are services associated with the same type of account. If you choose to delete your Mercado Libre account, your Mercado Pago account will be deleted too.",
```
Comment (Nightfirecat) [inline _data/sites.json]: `</br>` isn't valid HTML. Let's switch these all to `<br>`

```suggestion
        "notes": "To cancel/delete your account, access the account deletion link specified above and log in to your account. In the options, select a reason why you want to cancel your account, click in the checkbox \"Estoy de acuerdo\", and confirm your action by clinking \"Aceptar\".<br><br><strong>Warning:</strong> You may cancel your account only if there is no limitation, partial and/or total blockage applied to it. You will also be unable to cancel your account if you have outstanding payments and/or purchases. Mercado Libre and Mercado Pago are services associated with the same type of account. If you choose to delete your Mercado Libre account, your Mercado Pago account will be deleted too.",
```

**Comment by SnwMds** [comment]
Oh, I didn't know that forcing push changes to this branch will result in a closed pull request. My mistake. Sorry for that.

I will open a new pull request with the requested changes now.

**Comment by tupaschoal** [comment]
It shouldn't, I have done it extensively on #333. If you explain what you did we can compare :P

**Comment by SnwMds** [comment]
Certainly, it was something I did wrong. I will perform some tests and find out how I can improve this. Git is still something new for me. There is so much I need to learn.

**Comment by tupaschoal** [comment]
Don't worry, I'm frequently caught by its traps too ;p

**Pull request closed without merging**